config/v1/0000_00_cluster-version-operator_01_clusterversion.crd.yaml

```diff
@@ -63,7 +63,7 @@ spec:
                   type: object
                   properties:
                     force:
-                      description: "force allows an administrator to update to an image that has failed verification, does not appear in the availableUpdates list, or otherwise would be blocked by normal protections on update. This option should only be used when the authenticity of the provided image has been verified out of band because the provided image will run with full administrative access to the cluster. Do not use this flag with images that comes from unknown or potentially malicious sources. \n This flag does not override other forms of consistency checking that are required before a new update is deployed."
+                      description: force allows an administrator to update to an image that has failed verification or upgradeable checks. This option should only be used when the authenticity of the provided image has been verified out of band because the provided image will run with full administrative access to the cluster. Do not use this flag with images that comes from unknown or potentially malicious sources.
```

Comment:
@wking  "upgradeable checks" is still somewhat unclear to someone not steeped in our internal meanings.

"there is no edge from your cluster version to this target version" is an "upgradeable check"(at least to a naive user) but not one that force has any influence on, right?